### PR TITLE
Docs/readme url

### PR DESCRIPTION
**Pull request by jiho-kr**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [x] Documentation content changes
-   [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Fix wrong contributors url

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Comment by Pigrabbit** [comment]
Would you please check merge conflict?

**Comment by jiho-kr** [comment]
thanks, i checked

File: README.md [hunk 1]
```diff
@@ -2,6 +2,53 @@
 
 A NestJS module for managing environment variables easily and securely.
 
+<hr/>
+
+## What Difference From [official library](https://docs.nestjs.com/techniques/configuration)
```
Comment (Pigrabbit) [inline README.md:7]: It seems like it doesn't resolved well 🤔
Currently, this section is under `## Features`.

Comment (jiho-kr) [reply README.md]: oops!. sorry, could you please check at it again?

**Review by Pigrabbit — APPROVED** [review]
Perfect! 👍🏼

**Pull request merged**